### PR TITLE
Add documentation about running federation tests

### LINKED ISSUE #2906: Missing docs about running federation tests
I see that there's an env variable REMOTE_BACKEND_URL, so maybe one needs to manually setup another OC instance there with the testing app enabled ?

Once figured out, we should update README.md to include the relevant information.

### PR DESCRIPTION
**Pull request by jasson99**
## Description
Add documentation about running federation tests

## Related Issue

- Fixes #2906 

## Motivation and Context
It is possible to run federation tests in owncloud. However, documentation about running federation tests is not available in the ownCloud README file.This PR adds documentation for running federation tests in ownCloud.

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,6 +94,12 @@ In the local Phoenix checkout, copy the `config.json.sample` file to `config.jso
     3. install the Chrome browser and let yarn start & run selenium (remember to set `LOCAL_UPLOAD_DIR` though)
     4. Use standalone selenium server (remember to set `SELENIUM_HOST`, `SELENIUM_PORT` and `LOCAL_UPLOAD_DIR`).
 - run `yarn run acceptance-tests <feature-files-to-test>`
+- to run federation tests, follow the steps below:
```
Comment (individual-it) [inline README.md]:
```suggestion
- to run federation tests:
```

File: README.md [hunk 2]
```diff
@@ -94,6 +94,12 @@ In the local Phoenix checkout, copy the `config.json.sample` file to `config.jso
     3. install the Chrome browser and let yarn start & run selenium (remember to set `LOCAL_UPLOAD_DIR` though)
     4. Use standalone selenium server (remember to set `SELENIUM_HOST`, `SELENIUM_PORT` and `LOCAL_UPLOAD_DIR`).
 - run `yarn run acceptance-tests <feature-files-to-test>`
+- to run federation tests, follow the steps below:                         
+   1. Install and setup a remote ownCloud server with different name than that of already installed local ownCloud server.
```
Comment (individual-it) [inline README.md]:
```suggestion
   1. Install and setup a second ownCloud server-instance that is accessible by a different URL. That second server must have its own database and data directory.
```

File: README.md [hunk 3]
```diff
@@ -94,6 +94,12 @@ In the local Phoenix checkout, copy the `config.json.sample` file to `config.jso
...
+   
+   2. clone and install testing app into remote ownCloud server from http://github.com/owncloud/testing .
```
Comment (individual-it) [inline README.md]:
```suggestion
   2. clone and install testing app into the second ownCloud server-instance from http://github.com/owncloud/testing .
```

File: README.md [hunk 4]
```diff
@@ -94,6 +94,12 @@ In the local Phoenix checkout, copy the `config.json.sample` file to `config.jso
...
+
+   3. run `REMOTE_BACKEND_HOST=http://<ip_address_of_remote_ownCloud_server> yarn run acceptance-tests <feature-files-to-test>` .
```
Comment (individual-it) [inline README.md]:
```suggestion
   3. when running the acceptance tests use  `REMOTE_BACKEND_HOST` env. variable to define its address e.g. `REMOTE_BACKEND_HOST=http://<ip_address_of_seconc_ownCloud_server> yarn run acceptance-tests <feature-files-to-test>` .
```

File: README.md (hunk 1 above)
Comment (jasson99) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (jasson99) [reply README.md]: fixed

File: README.md (hunk 3 above)
Comment (jasson99) [reply README.md]: fixed

File: README.md (hunk 4 above)
Comment (jasson99) [reply README.md]: fixed

**Comment by PVince81** [comment]
Looking at the drone file, it seems there's more to it, like setting the skeleton folder:
```
			'cd /var/www/owncloud/federated/',
			'php occ a:e testing',
			'php occ config:system:set trusted_domains 2 --value=federated',
			'php occ log:manage --level %s' % logLevel,
			'php occ config:list',
			'php occ config:system:set skeletondirectory --value=/var/www/owncloud/federated/apps/testing/data/webUISkeleton',
			'php occ config:system:set dav.enable.tech_preview  --type=boolean --value=true',
			'php occ config:system:set sharing.federation.allowHttpFallback --value=true --type=bool'
```

I tried this PR's suggestion and the tests are failing for me with:
```
  Scenario: sharing indicator of items inside a re-shared folder
    ✖ failed
      Error: Could not set skeletondirectory. Status:500 Internal Server Error
          at Object.checkStatus (/home/vincent/Private/Work/workspace/phoenix/tests/acceptance/helpers/httpHelper.js:40:11)
          at _callee$ (/home/vincent/Private/Work/workspace/phoenix/tests/acceptance/helpers/config.js:31:14)
          at tryCatch (/home/vincent/Private/Work/workspace/phoenix/node_modules/regenerator-runtime/runtime.js:45:40)
          at Generator.invoke [as _invoke] (/home/vincent/Private/Work/workspace/phoenix/node_modules/regenerator-runtime/runtime.js:271:22)                                                                                                                                  
          at Generator.prototype.(anonymous function) [as next] (/home/vincent/Private/Work/workspace/phoenix/node_modules/regenerator-runtime/runtime.js:97:21)                                                                                                              
          at asyncGeneratorStep (/home/vincent/Private/Work/workspace/phoenix/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)                                                                                                                                   
          at _next (/home/vincent/Private/Work/workspace/phoenix/node_modules/@babel/runtime/helpers/asyncToGenerator.js:25:9)
          at process._tickCallback (internal/process/next_tick.js:68:7)
```
but no log entries

**Comment by PVince81** [comment]
hmm, same problem even after setting the above occ settings

**Comment by PVince81** [comment]
never mind, I forgot to enable the testing app on the remote.

**Pull request merged**